impl/client/src/main/javascript/web/dojo/pentaho/common/TemplatePicker.html

```diff
@@ -20,7 +20,47 @@
                         </td>
                         <td>
                             <div style="overflow:auto; width:100%; height: 100%">
-                                <table data-dojo-attach-point="templateListTbl" border="0" cellspacing="0" cellpadding="0" style="width:100%">
+                                <!--#region responsive implementation-->
```

Comment:
Ideally, the dialog would have only one set of markup. The CSS style sheets of responsive themes (ruby) would make that otherwise static markup become responsive.

Having two sets of markup, non-responsive and responsive, looks to me like a last resort solution. It has a much higher maintenance cost compared to the alternative.

I would like to better understand the reasons for adding the new set of markup. What issues were found trying to make the existing markup responsive (we have been able to make tables responsive in other instances). Alternatively, one could try to update all non-responsive supported themes to support the new markup.

Lastly, in the proposed implementation, the responsive behavior is implemented in PIR only (`pirRuby.css`), but it should be in `globalRuby.css` matching the _scope_ of this class.

Reply:
As we discussed on call, I have made/tested the following changes:
- Keep the new HTML
- Get rid of the old HTML
- Cleanup js and HTML
- Move the new CSS from pirRuby.css to globalRuby.css
- Copy new CSS over to old themes
https://github.com/pentaho/pentaho-commons-gwt-modules/pull/992